.chloggen/oracledb-stored-procedure-info.yaml

```
@@ -0,0 +1,30 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. receiver/filelog)
+component: receiver/oracledb
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add stored procedure information (procedure_id, procedure_name, procedure_type) to Oracle DB receiver logs for top queries and query samples.
```

```suggestion
note: Add stored procedure information to logs for top queries and query samples.
```
Nit: The `subtext` is enough to provide more information here.